cfme/tests/services/test_myservice.py

```diff
@@ -45,7 +46,9 @@ def myservice(setup_provider, provider, catalog_item, request):
     Metadata:
         test_flag: provision
     """
-    vm_name = catalog_item.provisioning_data["vm_name"] + "_0001"
+    vm_name = version.pick({
+        version.LOWEST: catalog_item.provisioning_data["vm_name"] + '_0001',
+        '5.7': catalog_item.provisioning_data["vm_name"] + '0001'})
```

Comment:
Can we just use the new syntax for earlier versions, or does using '_0001' in 5.6 do something worthwhile for us?